qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferDbConnector.groovy

```diff
@@ -104,4 +114,78 @@ class OfferDbConnector implements CreateOfferDataSource{
             return generatedKeys[0]
         }
     }
+
+    List<OfferOverview> loadOfferOverview() {
+        List<OfferOverview> offerOverviewList = []
+
+        String query = "SELECT modificationDate, projectTitle, " +
+                "totalPrice, first_name, last_name, email\n" +
+                "FROM offer \n" +
+                "LEFT JOIN person \n" +
+                "ON offer.customerId = person.id"
+
+        Connection connection = connectionProvider.connect()
+        connection.withCloseable {
+            PreparedStatement statement = it.prepareStatement(query)
+            ResultSet resultSet = statement.executeQuery()
+            while (resultSet.next()) {
+                def customer = new Customer.Builder(
```

Comment:
I'd be more specific here to increase readability
```suggestion
                Customer customer = new Customer.Builder(
```

Reply:
Yeah, this is a general thing we need to talk in the developer round. This declaration style is Groovy, and enhances readability. Moreover, the Groovy compiler allows for dynamic typing, which we can use here.

I would argue, that from the Builder it is clear, which object is returned here.

Reply:
I'd agree that some sort of standardization makes sense for future development. I mostly noticed it since there are unused imports if we go for more groovy-esque typing(ProjectManager)